### PR TITLE
Upgrade to Plone 5.1

### PR DESCRIPTION
**Pull request by Netroxen**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
 *.egg-info
 *.log
 *.mo
 *.py?
 *.swp
-
-
-# Dirs
-
+# dirs
```
Comment (santonelli) [inline .gitignore]: What exactly is the benefit for the migration to rename a comment?

File: .travis.yml [hunk 2]
```diff
@@ -1,26 +1,27 @@
...
-  firefox: latest-esr
+  - downloads
+python:
+  - 2.7
+matrix:
+  fast_finish: true
 install:
-- python bootstrap-buildout.py
-- bin/buildout -c travis.cfg
+  - python buildout.py
```
Comment (santonelli) [inline .travis.yml]: Why do you rename this? Is it possible to get rid of the file entirely and use ./bin/pip install -r requirements instead?

File: setup.py [hunk 3]
```diff
@@ -14,14 +14,13 @@
 
 setup(
     name='operun.crm',
-    version='2.1.1',
-    description="operun CRM",
+    version='2.1.0',
```
Comment (santonelli) [inline setup.py]: What?

File: src/operun/crm/configure.zcml [hunk 4]
```diff
@@ -7,17 +7,15 @@
     xmlns:plone="http://namespaces.plone.org/plone"
     i18n_domain="operun.crm">
 
-  <!-- Defaults -->
```
Comment (santonelli) [inline src/operun/crm/configure.zcml]: Is it necessary to touch this file?

File: .travis.yml (hunk 2 above)
Comment (Netroxen) [reply .travis.yml]: `./bin/pip install -r requirements.txt` I tried this but the Travis CI messed up, not sure why so for now I keep just the default how the package came with it.

File: .gitignore (hunk 1 above)
Comment (Netroxen) [reply .gitignore]: Copied the .gitignore from the default bobtemplates output, who knows. Maybe THEY fixed that.

**Pull request merged**